### ISSUE TITLE
FE | Profile Hub | Build the Prototype in Staging

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#64160]
## Background 
Using the mocks from design intent we should build out a prototype to use during our [usability research study](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/Research/2023-09-profile-hub-usability)

We will test WITHOUT a nav menu on both mobile and desktop. 

## Updated hi-fi mock-ups
-  User flow is available on [the mobile page](https://sketch.com/s/b41a687f-8ab8-4be1-b189-692e5818cffd) 
- [Mobile](https://sketch.com/s/b41a687f-8ab8-4be1-b189-692e5818cffd)
- [Desktop](https://www.sketch.com/s/b41a687f-8ab8-4be1-b189-692e5818cffd/p/EFCD1C39-A0C1-4A07-8510-1C0A7713FA1A/canvas)

## Edge Cases to incorporate 
- we need to incorporate the Bad Address Indicator alert that design worked through [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/64818), see the [user flow sketch file with the BAI called out](https://www.sketch.com/s/b41a687f-8ab8-4be1-b189-692e5818cffd)

## Tasks 
- [x] The UI matches the mock
- [x] All links are present and route to appropriate page
- [x] Mobile styling matches mock design
- [ ] ~~Confirmed that link analytics events are being fired.~~ separate ticket needed
- [x] Confirm that page is gated by any restrictions: fiduciary, deceased, incompetent indicators

## Acceptance Criteria 
- [ ] The UI matches the mock
- [ ] PM and Design have checked things and feel ready for Usability Testing! #63621 #63622

**Comment by andaleliz** [comment]
@adamwhitlock1 I don't know that we need to think about this for usability testing BUT I realized that we used to use the personal info page as a place for important alerts that impact other parts of profile. For example, the [Bad Address indicator alert](https://www.sketch.com/s/59857eb5-d9f9-4145-99d3-d9a1de2d0655/a/JnRG2qz).  That should probably get moved to the profile hub if that's the new entry point for profile, and we haven't accounted for alerts on that page yet.

When is a good time to start thinking through those use cases? Do you know if there are other alerts that display on the personal info page that are actually about other parts of profile?

**Comment by adamwhitlock1** [comment]
There aren't any other alerts that would appear on the personal info page that should move over to the Hub page, but yea that is a good call to make sure the BAI alert is present on the HUB when needed.

This sprint I'm getting the foundation put together for adding the Hub and next sprint is when i should have the full Hub up in staging, so at that point is probably when I would add the alert and tie in the logic for it.

**Comment by andaleliz** [comment]
@mtcA6 I created [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/64818) to cover what was discussed above. I think this might need to be "unplanned work" in this sprint so we can have it read for Adam when he gets to the point of being able to work on this.  

If @pamela-drouin is able to kick off recruiting for the profile hub in this sprint, this could be a good "down time" task in the 8 days we have to allow Perigean. I also have bandwidth to work on this.

**Comment by mtcA6** [comment]
@andaleliz ok i've tagged it in that unplanned work bucket on the sprint ticket. I don't know that we'll be able to recruit by end of this sprint, I'm thinking next sprint because its looking like the Hub FE tickets likely wont be fully done until the end of next sprint.

**Comment by andaleliz** [comment]
Hi @adamwhitlock1 just an FYI I updated the links in the ticket description to point to the hi-fi mock-ups instead of the wireframes, since they exist now.

Also, Pam caught a pair of links that weren't ordered correctly. In the Account security card, the "review account security" link should come first.  This is reflected in the mock-ups now.

**Comment by andaleliz** [comment]
@mtcA6 what is the plan for analytics tracking? The info in the ticket description is pretty light. I see in the [product outline ](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/hub#analytics)it says “We need new tracking for GA on the profile menu" but there isn't a profile menu on the hub so I'm not quite sure what that's referring to.

**Comment by mtcA6** [comment]
@andaleliz I think when I wrote that, I don't believe we had made the decision to remove the menu for both mobile and desktop, that was still in flux. In my mind, we would have the old nav items and then new links from the cards. Since then, we've decided to run usability testing WITHOUT the nav menu for mobile or desktop. 

That said, I'm not certain we need to make a hard decision on the path to take for GA until after the usability research. Unless you need events firing for the usability testing. 

I saw you mentioned GA in the team channel as well, I'll schedule a call so we can discuss this 

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/21df62df-f30d-4d87-9f8b-62ea1671c7aa)

cc: @Samara-Strauss

**Comment by andaleliz** [comment]
@mtcA6 nope, no need to have GA events for usability testing. We can figure it out later. I saw analytics as part of this ticket and mentioned in the outline, so thought it was further along than it is.

We don't need to have a sync - my reference to GA in Slack is exactly what I brought up in this ticket.

**Comment by adamwhitlock1** [comment]
Last PR hopefully for this ticket is in.

https://github.com/department-of-veterans-affairs/vets-website/pull/25877

It gates the page for specific flags and adds tests for the blocked cases along with bad address indicator E2E tests

If we roll this over to next sprint its at most a point left of work for any revisions (or maybe just a new work for any tweaks that come from reviewing it further)

**Comment by mtcA6** [comment]
created #66403
